### PR TITLE
changed overflow behavior for My Files list to make the dropdown stay…

### LINKED ISSUE #1291: Dropdown inside My Files list renders beneath containing frame
As noted by @mtribone in #1282, ```overflow-x: auto``` causes the "Select an Action" dropdown to be rendered beneath the containing frame (in both Firefox and Chrome).  #1282 was merged because without this change, the 6.3.0 release caused more severe display problems for Firefox users.  However, the fix implemented in #1282 creates this problem, which should still be addressed.

9299c994-4657-11e5-8878-24ccbccf011a

### PR DESCRIPTION
**Pull request by StudioZut**
… visible and play nice with firefox 
Fixes #1291

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Thoughts, @mtribone @jcoyne @cam156 ?

**Comment by kerchner** [comment]
@StudioZut and I are perplexed as to how this PR's change to  ```app/assets/stylesheets/sufia/_file-listing.scss```  would have decreased coverage of  ```/sufia-models/app/models/concerns/sufia/generic_file/full_text_indexing.rb``` (according to Coveralls).  Any insight would be welcomed.

**Comment by mjgiarlo** [comment]
@kerchner That happens sometimes. We ignore it. ;)  No worries.

File: app/assets/stylesheets/sufia/_file-listing.scss [hunk 1]
```diff
@@ -5,7 +5,10 @@ h4 .small {
   color: $navbar-default-toggle-icon-bar-bg;
 }
 
-@media screen and (max-width: 767px) {
+@media screen and (max-width: 961px) {
+  #documents.table-responsive {
```
Comment (mtribone) [inline app/assets/stylesheets/sufia/_file-listing.scss]: Not a blocker, but I was wondering if this declaration works without the specificity of using the ID #documents? Using ID selectors in CSS makes them hard to override. More of a style comment.

File: app/assets/stylesheets/sufia/_file-listing.scss [hunk 2]
```diff
@@ -170,6 +178,10 @@ input#check_all {
   margin-bottom: 10px;
 }
 
+#appliedParams.constraints-container {
```
Comment (mtribone) [inline app/assets/stylesheets/sufia/_file-listing.scss]: Same comment about using ID selectors as above. Also, I am unsure where this is located or what it controls. Where is this used?

File: app/assets/stylesheets/sufia/_file-listing.scss [hunk 3]
```diff
@@ -112,6 +115,11 @@ h4 .small {
 .sorts-dash, .sorts, .expandable_new {
   cursor: pointer;
 }
+
+.batch-info .sort-toggle {
```
Comment (mtribone) [inline app/assets/stylesheets/sufia/_file-listing.scss]: I'm guessing this addresses the visual gap between the sort controls and the file listing?

**Comment by mtribone** [comment]
Just skimmed some of the style sheets and found 5 occurrences of #documents. Looks like I've been asleep at the wheel. LOL.

File: app/assets/stylesheets/sufia/_file-listing.scss (hunk 3 above)
Comment (StudioZut) [inline app/assets/stylesheets/sufia/_file-listing.scss]: Yes, that was my best effort at solving that

File: app/assets/stylesheets/sufia/_file-listing.scss (hunk 1 above)
Comment (StudioZut) [inline app/assets/stylesheets/sufia/_file-listing.scss]: This was me being cautious and trying to limit the impact of my changes.

File: app/assets/stylesheets/sufia/_file-listing.scss (hunk 2 above)
Comment (StudioZut) [inline app/assets/stylesheets/sufia/_file-listing.scss]: The left and right border were extending below the bottom border

Comment (StudioZut) [inline app/assets/stylesheets/sufia/_file-listing.scss]: And the ID selector is like the other one, didn't want to bork anything else that used the class

**Pull request merged**